lib/quartz/go_process.rb

```diff
@@ -6,8 +6,12 @@ def self.processes
     @processes ||= []
   end
 
+  def seed
+    @seed ||= rand(10000)
```

Comment:
pong
It's great Idea, I had no idea for atomic operand and thanks for notice.
I will commit for this!